models/tasks.js

```diff
@@ -86,8 +90,11 @@ const fetchActiveTaskMembers = async () => {
 const fetchTask = async (taskId) => {
   try {
     const task = await tasksModel.doc(taskId).get()
-    const taskData = task.data()
-    return { taskData: await fromFirestoreData(taskData) }
+    const taskData = await fromFirestoreData(task.data())
+    if (taskData?.status) {
+      taskData.status = TASK_STATUS[taskData.status]
```

Comment:
Currently, we have task status as **pending** and **High Priority** please those cases as well